services/validators/envValidator.go

```diff
@@ -151,6 +151,60 @@ func closeDBConnection(handle *sql.DB) {
 	}
 }
 
+func killDanglingDBConnections(db *sql.DB) {
+
+	rows, err := db.Query(`SELECT PID, QUERY_START, WAIT_EVENT_TYPE, WAIT_EVENT, STATE, QUERY
+							FROM PG_STAT_ACTIVITY
+							WHERE PID <> PG_BACKEND_PID()
+							AND APPLICATION_NAME = (
+								SELECT APPLICATION_NAME FROM PG_STAT_ACTIVITY WHERE PID = PG_BACKEND_PID()
```

Comment:
```suggestion
								SELECT APPLICATION_NAME FROM PG_STAT_ACTIVITY WHERE PID = PG_BACKEND_PID()
								current_setting('application_name');
```

Reply:
Just wanted to understand how would this query behave in case of existing multi tenant customers since all of them share the postgres(RDS).Should we pass a unique application name for each rudder-server or just a single one would serve the purpose?

Reply:
We should not use same application name across deployments or even for different nodes in the same deployment. Using Hostname should be ok as our deployment is always done on kubernetes and each rudder server will run in it's own pod and pod name becomes the host name. In short, we shouldn't set this env at all in production. Exposing it as env configuration can still be debated for some use cases.

Reply:
Reuse instanceId for this?

Reply:
So setting `application_name` as an ENV is not recommended and we should be instead using the pod name as application name right

Reply:
> One edge case to think about. If for some reason our service is connected with no `APPLICATION_NAME`, could be due to a proxy or a misconfiguration. This query will match all connections from an application without name.
> 
> I would add a safeguard here, that

I don't see how a hostname can be empty, nevertheless will include the safeguard you propose

Reply:
> So setting `application_name` as an ENV is not recommended and we should be instead using the pod name as application name right

Yes, I shall remove support for overriding the application name through an environment variable